.github/workflows/qat.yml

```diff
@@ -1,89 +1,75 @@
-name: QAT
+name: 'Checkly QAT'
 
 on:
   workflow_call:
     inputs:
       QA_SITE_DOMAIN:
         default: ''
         description: 'QA Site Domain'
-        required: false
+        required: true
         type: string
       THEME_NAME:
         default: ''
         description: 'WP Theme'
-        required: false
+        required: true
         type: string
 
     secrets:
-      BITBUCKET_READ_ONLY_SSH_KEY:
```

Comment:
Both this and the `GITHUB_READ_ONLY_SSH_KEY` entries must stay, but can be marked as not required. Every theme repo (and `pmc-plugins`) passes these and if they're removed from here, the workflows will fail.